src/V3AstNodeDType.h

```diff
@@ -793,6 +793,10 @@ class AstEnumDType final : public AstNodeDType {
         dtypep(nullptr);  // V3Width will resolve
         widthFromSub(subDTypep());
     }
+    AstEnumDType(const AstEnumDType& other)
+        : AstNodeDType(other)
+        , m_name(other.m_name)
+        , m_uniqueNum(uniqueNumInc()) {}
```

Comment:
There is similar code in 3 other places - presumably all should be handled similarly?